### ISSUE TITLE
Shortcut for "Rename" operates on wrong line / address.

### ISSUE THREAD

**Issue by Semnodime** [rizinorg/cutter#2631]
**Environment information**
* Operating System: linuxmint 20.1
* Cutter version: 1.12.0
* File format: any
* Arch: any
* Type: any

**Describe the bug**
Shortcut for "Rename" operates on wrong line / address after navigating via the goto bar.

**To Reproduce**
1. Select top most line of function in disassembly tab
2. Rename via shortcut <kbd>N</kbd>
3. Press <kbd>g</kbd>, type address of another function start, press <kbd>Enter</kbd>
4. Press <kbd>N</kbd> to rename this function.
5. Observe the dialog wrongly referencing the old address

**Expected behavior**
Dialog for renaming the currently selected address.

**Screenshots**
Notice how (after the goto via the navigation bar) the right address is selected, but the shortcut / dialog references the old address.
![image](https://user-images.githubusercontent.com/19511852/110663612-e460f800-81c6-11eb-8818-2985521590ad.png)

**Comment by ITAYC0HEN** [comment]
Does it happen with other keyboard shortcuts as well? e.g comment

**Comment by karliss** [comment]
Guessing the mechanism of problem:
a) disassembly context menu offset is not updated properly on specific navigation event -> should affect everything in disassembly context menu
b) rename action does something when opening -> the problem should also affect it when triggering using shortcut after mouseclick

**Comment by Semnodime** [comment]
> Does it happen with other keyboard shortcuts as well? e.g comment
Nope, the problem does _not_ apply to comments. A comment will get attached / edited for the correct line, while the rename still targets the last valid item.
